src/helpers/hendleFilteredTodos.ts

```diff
@@ -0,0 +1,16 @@
+import { FilterOptions } from '../enums/FilterOptions';
+import { Todo } from '../types/Todo';
+
+export const hendleFilteredTodos = (
```

Comment:
Let's use more descriptive name for this helper  
```suggestion
export const getFilteredTodos = (
```